torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16832,8 +16832,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            skips=(
            )),
     OpInfo('unique',
-           dtypes=all_types_and(torch.bool, torch.float16, torch.bfloat16),
-           dtypesIfCUDA=all_types_and(torch.bool, torch.float16),
+           dtypes=all_types_and(torch.bool, torch.float16, torch.bfloat16, torch.uint16, torch.uint32, torch.uint64),
+           dtypesIfCUDA=all_types_and(torch.bool, torch.float16, torch.uint16, torch.uint32, torch.uint64),
```

Comment:
torch.uint16 exists?

Reply:
yeah! It doesn't support that many ops but we're happy to add them as requested

Reply:
It's mostly useful with torch.compile, which has fairly complete compilation coverage for it.